general_autoencoder.py

```diff
@@ -292,23 +365,30 @@ def fill_feed_dict(self, data, regularization_weighting, ages=None, idxs=None):
                 indexed_ages = ages[idxs]
             else:
                 indexed_ages = ages
```

Comment:
This was pretty confusing to me (maybe I even wrote this line). Could we change it to `indexed_ages = None`?

Reply:
i agree it's confusing. i don't think we can just change it to None? basically, I think all the variables prefixed with indexed_ really mean (semantically) "the version of the variable you want to use". so perhaps we could make it clearer by changing the variable name: eg, indexed_ages -> ages_to_use -- and adding a comment. does that sound reasonable?

Reply:
oh sorry. i think I see what you're saying. yes, we can do that.

Reply:
i implemented these changes and also renamed the variables + commented to hopefully clarify.